inference/enterprise/parallel/parallel_http_config.py

```diff
@@ -14,13 +14,7 @@
 if REDIS_HOST is None:
     raise RuntimeError("Redis must be configured to use async inference")
 pool = ConnectionPool(host=REDIS_HOST, port=REDIS_PORT, decode_responses=True)
-checker = ResultsChecker(
-    AsyncRedis(
-        host=REDIS_HOST, port=REDIS_PORT, decode_responses=True, max_connections=1000
-    )
-)
-model_manager = DispatchModelManager(model_registry, checker)
-model_manager.init_pingback()
+model_manager = None
```

Comment:
Gross hack for now: initializing this stuff in the on app startup to make sure that model manager and web server are using same event loop